Georgian.Employee.ResponsiveUI/ResponsiveUI/GcResponsive/GC_responsive_ui.css

```diff
@@ -27,13 +27,13 @@ body {
 }
 
 .ms-rtePosition-2, .ms-rtePosition-1 {
-    width: 100%;
+    /*width: 100%;*/
```

Comment:
Please use comments for helping understand the reason for the code/style and let the version control system help us understand the code history.

This line should be removed.